whelk-core/src/main/groovy/whelk/converter/marc/NormalizeContentTypeStep.groovy

```diff
@@ -0,0 +1,27 @@
+package whelk.converter.marc
+
+class NormalizeContentTypeStep extends MarcFramePostProcStepBase {
+    def contentType = 'contentType'
+    def shape = [TYPE, contentType] as Set
+
+    void modify(Map record, Map thing) {
+        moveContentType(thing)
+    }
+
+    boolean moveContentType(Map thing) {
+        def work = thing.instanceOf
+        def moved = work?.hasPart?.removeAll { p ->
+            if (p.keySet() == shape) {
```

Comment:
In theory, someone could add a part with a more specific type than `Work`, which if different from the parent work could indicate that this is a proper part. (We should check this assumption though.)

Reply:
I found only one example where the type is not `Work` (https://libris.kb.se/katalogisering/w89srsd1t3wbnhzp). It looks like the part was added manually, however I can't tell why that needs to be a separate part. I guess no need to address this now since we have no deviating records from import?

Reply:
Yes, of course, you're absolutely right! This is only done for data coming from MARC, so at this point we know the type is only `Work`! Thus this is perfectly fine!